frontend/src/plugins/impl/anywidget/AnyWidgetPlugin.tsx

```diff
@@ -1,6 +1,7 @@
 /* Copyright 2024 Marimo. All rights reserved. */
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import { z } from "zod";
+import type { AnyWidget, AnyModel, Initialize, Render } from "@anywidget/types";
```

Comment:
Not sure if it would be useful to use these types directly, I realize the AnyModel interface is a little tricky to faithfully implement with all the overloads for various methods.

We could consider exporting something less type safe (but easier to implement) for implementers. The strict type returns are mostly useful for developers implementing widgets.

Reply:
Would it be possible to export just types? I noticed this pulls in a bunch of old deps (`jquery` and `backbone`)

```
  /@jupyter-widgets/base@6.0.8(react@18.3.1):
    resolution: {integrity: sha512-2Y0gl6BXdlxU9DR4YBWXd3dv9m6kSd2T31W81XGLR5/MGH4SSZ2LuiC0TsuMRFL+b6dk4+X73tRaekRxo5iMKA==}
    dependencies:
      '@jupyterlab/services': 7.2.4(react@18.3.1)
      '@lumino/coreutils': 2.2.0
      '@lumino/messaging': 1.10.3
      '@lumino/widgets': 2.4.0
      '@types/backbone': 1.4.14
      '@types/lodash': 4.14.194
      backbone: 1.4.0
      jquery: 3.7.1
      lodash: 4.17.21
```

Reply:
I think that comes from the widget manager. I'll work on dropping that dep and making a more minimal interface.

Reply:
for now, i will just copy over the necessary types. 

do you mind singing the CLA too so builds pass? (just need to comment back)

Reply:
Sounds good. I'll work on cleaning up @anywidget/types separately!